src/Microsoft.AspNetCore.Authentication.Facebook/FacebookOptions.cs

```diff
@@ -57,6 +60,6 @@ public string AppSecret
         /// The list of fields to retrieve from the UserInformationEndpoint.
         /// https://developers.facebook.com/docs/graph-api/reference/user
         /// </summary>
-        public IList<string> Fields { get; }
+        public ICollection<string> Fields { get; } = new HashSet<string>();
```

Comment:
Do you see senarios other than add remove count for each?
IColection seems to provide enough but I'll see what @Tratcher thinks. It's an easy change.
Re scope yes I agree but you'll have to have that discussion with @Tratcher